src/common/components/repository-row/index.js

```diff
@@ -276,27 +275,15 @@ export class RepositoryRowView extends Component {
 
     return (
       <Row isActive={isActive}>
-        <Data col="27">
-          { hasBuilt
-            ? (
-              <Link to={ `/user/${fullName}` }>{ fullName }</Link>
-            )
-            : (
-              <span>{ fullName }</span>
-            )
-          }
-        </Data>
-        <Data col="15">
-          { this.renderConfiguredStatus(snap) }
-        </Data>
-        <Data col="25">
-          { this.renderSnapName(registeredName, showRegisterNameInput, snap) }
-        </Data>
-        <Data col="30">
+        <DataLink col="28" to={ hasBuilt ? `/user/${fullName}` : null }>
+          <strong>{ fullName }</strong>
+        </DataLink>
+        { this.renderConfiguredStatus(snap) }
+        { this.renderSnapName(registeredName, showRegisterNameInput, snap) }
+        <DataLink col="24" to={ hasBuilt ? `/user/${fullName}` : null }>
```

Comment:
This could use some cleaning up (moving all cells to separate functions - or even small components).
Not sure if it's worth touching now...

Thoughts?